index.html

```diff
@@ -144,22 +202,20 @@ <h5>About Myself</h5>
     </section>
         <!-- contact me  -->
     <section class="contact">
-        <div class="contact-bg-img">
-            <h6>Contact me</h6>
-            <p class="contact-p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
-            I’d love to help with it!</p>
-            <form action="https://formspree.io/f/xjvdador" method="post">
-                <label for="name">Name:</label>
-                <input type="name" id="name" name="user_name" placeholder="Lorem Ipsum" maxlength="30" required>
-                <label for="email">Email:</label>
-                <input type="email" id="email" name="user_name" placeholder="loremipsum@gmail.com" required>
-                <label for="msg">Message:</label>
-                <textarea name="user_name" id="msg" cols="15" rows="10" maxlength="500" required>Write your message here...</textarea>
-                <button class="contact-mobile-btn" type="submit">Get in touch</button>
-            </form>
-        </div>
-        <div class="home-btn">
-        </div>
+        <div class="contact-bg-img"></div>
+                <h6>Contact me</h6>
```

Comment:
- [ ] please kindly make the contact section full width and also add the background image so it follows the Figma design.
- [ ] please do not make the text in the contact section bold, as it is lighter than the contact me heading.
yours
![Screenshot (1235)_LI](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/65298048/4afdfd6d-0218-451e-8f9b-f1f51017b89c)
figma
![Screenshot (1236)](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/65298048/cf392fd6-79b6-49f5-b768-0d68fc1b85d8)